### DISCUSSION TITLE
Recipe Books

### DISCUSSION THREAD

**Discussion by cmr9082** [AngryBeaver/beavers-crafting#78] [Ideas]
I love the many individual recipes, but I wish there was a way we could compile them. And if a player had access to the book they also would have access to the many different recipes the book references instead of needing to possess a ton of recipe sheets.

**Comment by AngryBeaver** [comment]
Well have to think about this. 
It would be nice if it fits to the current folder input field so you can toggle all recipes of a book.
You might then also start organizing the recipes in real folders one for each book and the recipe folder input will take this into account. 
But then each recipe can only be in one book -> a limitation.

Or you can reference recipes in one new book item this way you could have many books that will learn you the same recipe.
But then you would not be able to list all recipes of a book at least not how it is currently done.

anyway currently busy will have to wait.
